src/main/java/at/hannibal2/skyhanni/features/fishing/trophy/GeyserFishing.kt

```diff
@@ -27,7 +29,8 @@ class GeyserFishing {
 
     @SubscribeEvent(priority = EventPriority.LOW, receiveCanceled = true)
     fun onReceiveParticle(event: ReceiveParticleEvent) {
-        if (!shouldProcessParticles() || event.type != EnumParticleTypes.CLOUD) return
+        if (!shouldProcessParticles()) return
+        if (event.type != EnumParticleTypes.CLOUD || event.count != 15 || event.speed != 0.05f || event.offset != LorenzVec(0.1f, 0.6f, 0.1f)) return
```

Comment:
not really a problem but is there a reason you use floats here instead of doubles

Reply:
i just saw that in https://github.com/hannibal002/SkyHanni/blob/97ab3476c490b2b035d1dafebbc47e27c1ecc176/src/main/java/at/hannibal2/skyhanni/features/event/diana/GriffinBurrowParticleFinder.kt#L93 it used floats instead of doubles so i just went with that. i can change it if its needed

Reply:
> i just saw that in
> 
> https://github.com/hannibal002/SkyHanni/blob/97ab3476c490b2b035d1dafebbc47e27c1ecc176/src/main/java/at/hannibal2/skyhanni/features/event/diana/GriffinBurrowParticleFinder.kt#L93
> it used floats instead of doubles so i just went with that. i can change it if its needed

oh i meant in the lorenzVec, the other floats are fine

Reply:
oh, yeah no real reason why im using them there